jdaviz/core/template_mixin.py

```diff
@@ -1086,7 +1089,7 @@ def __init__(self, *args, **kwargs):
                                            'spatial_subset_items',
                                            'spatial_subset_selected',
                                            'spatial_subset_selected_has_subregions',
-                                           default_text='No Subset',
+                                           default_text='Entire Cube',
```

Comment:
In chatting with @kecnry, we realized we're effectively overriding the default `"No Subset"` label in both Model Fitting and Line Analysis. Rather than again overriding the default text in another plugin, @kecnry suggested we just change the mixin default directly.